CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Added the rendering of static members for a component.
 * Fixed security vulnerabilities.
+* Updated `jsonata` dependency
```

Comment:
please add this entry a [unreleased] section  and rename rntry to "Updated `jsonata` dependecy to version 2.x and refactored code to use async functions."